.github/actions/generate-commit-list-v1/src/doesBranchOrTagExist.test.ts

```diff
@@ -0,0 +1,75 @@
+import 'mocha'
+import { assert } from 'chai'
+import sinon from 'sinon'
+import * as exec from '@actions/exec'
+import doesBranchOrTagExist from './doesBranchOrTagExist'
+
+describe('doesBranchOrTagExist', () => {
+  let getExecOutputStub: sinon.SinonStub
+
+  beforeEach(() => {
+    getExecOutputStub = sinon.stub(exec, 'getExecOutput')
+  })
+
+  afterEach(() => {
+    getExecOutputStub.restore()
+  })
+
+  describe('when the branch exists', () => {
+    it('returns true', async () => {
```

Comment:
You should verify that the code is calling the git command with the correct input. Right now you only modify the stub return but if passing a branch actually called the tag code path this test would still pass even though that is incorrect. You'll need to do this for the tag parameter as well.